src/main/plugins/org.dita.pdf2/xsl/fo/loc.xsl

```diff
@@ -0,0 +1,174 @@
+<?xml version='1.0'?>
+
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+                xmlns:xs="http://www.w3.org/2001/XMLSchema"
+                xmlns:fo="http://www.w3.org/1999/XSL/Format"
+                xmlns:dita-ot="http://dita-ot.sourceforge.net/ns/201007/dita-ot"
+                xmlns:dita2xslfo="http://dita-ot.sourceforge.net/ns/200910/dita2xslfo"
+                xmlns:opentopic="http://www.idiominc.com/opentopic"
+                xmlns:opentopic-index="http://www.idiominc.com/opentopic/index"
+                xmlns:ot-placeholder="http://suite-sol.com/namespaces/ot-placeholder"
+                exclude-result-prefixes="opentopic opentopic-index dita2xslfo ot-placeholder xs"
+                version="2.0">
+
+  <!-- The variable change-items contains a list of all change-item elements that have a change-completed date. -->
+  <xsl:variable name="change-items" select="//*[contains (@class, ' relmgmt-d/change-item ')][*[contains (@class, ' relmgmt-d/change-completed ')] != '']"/>
+
+  <!-- Keep date format in a variable -->
+  <xsl:variable name="dateFormat"><xsl:call-template name="getVariable"><xsl:with-param name="id" select="'#date-format'"/></xsl:call-template></xsl:variable>
+
+  <!--   LOC   -->
+  <xsl:template match="ot-placeholder:changelist" name="createChangeList">
+    <xsl:if test="//*[contains(@class, ' relmgmt-d/change-historylist ')]">
+      <fo:page-sequence master-reference="toc-sequence" xsl:use-attribute-sets="page-sequence.loc">
+        <xsl:call-template name="insertTocStaticContents"/>
+        <fo:flow flow-name="xsl-region-body">
+          <fo:block start-indent="0in">
+            <xsl:call-template name="createLOCHeader"/>
+            <xsl:apply-templates select="//*[contains(@class, ' topic/critdates ')][1]"/>
+          </fo:block>
+        </fo:flow>
+      </fo:page-sequence>
+    </xsl:if>
+  </xsl:template>
+
+  <xsl:template name="createLOCHeader">
+    <fo:block xsl:use-attribute-sets="__lotf__heading" id="{$id.lot}">
+      <fo:marker marker-class-name="current-header">
+        <xsl:call-template name="getVariable">
+          <xsl:with-param name="id" select="'List of Changes'"/>
+        </xsl:call-template>
+      </fo:marker>
+      <xsl:call-template name="getVariable">
+        <xsl:with-param name="id" select="'List of Changes'"/>
+      </xsl:call-template>
+    </fo:block>
+  </xsl:template>
+  
+  <xsl:template match="*[contains(@class, ' bookmap/bookmeta ')]/*[contains(@class, ' topic/critdates ')]">
+
+    <!-- Create a list of all book releases -->
+    <xsl:variable name="bookReleaseDates" select="*[contains (@class, ' topic/created ')]/@date |
+                                                  *[contains (@class, ' topic/revised ')]/@modified"/>
+    <xsl:variable name="bookReleaseDateString">
+      <xsl:call-template name="getVariable">
+        <xsl:with-param name="id" select="'Book Release Date'"/>
```

Comment:
It would simplify the code to combine these into one `getVariable` call instead of using variable for the date + variable for colon + variable for separator + value of date.

In the variable file, you can refer to other variables, so the "Book Release Date" variable itself can be a combination of (date text) + colon + separator.

You can also pass the date in as a parameter when getting the "Book Release Date" variable -- so that you end up only getting + rendering a single OT variable, and never need to set `xsl:variable`:
```
<xsl:call-template name="getVariable">
  <xsl:with-param name="id" select="'Book Release Date'">
  <xsl:with-param name="params">
    <date><xsl:value-of select="format-date(., $dateFormat)"/></date>
  </xsl:with-param>
</xsl:call-template>
```

This will work with an updated string value:
```
<variable id="Book Release">Book Release<variable id="ColonSymbol"/><variable id="figure-number-separator"/><param ref-name="date"/></variable>
```